examples/models/exchange.yaml

```diff
@@ -1,5 +1,6 @@
 id: bitfinex
 name: Bitfinex
+platform: stellar, bitshares, ethereum:Ox
```

Comment:
I thought only one value would be used..
like 
`platform: stellar`
or `platform: ethereum:0x`